### ISSUE TITLE
DataStore for getting API key is down

### ISSUE THREAD

**Issue by jkropko** [propublica/congress-api-docs#284]
The datastore seems to be down: https://www.propublica.org/datastore/api/propublica-congress-api. I receive a message that says "An internal server error occurred."
Additionally, GET requests that previously worked for me are now returning 403 errors

**Comment by ariel-serruya** [comment]
Any progress on this? I was hoping to submit a new API key request form for commercial usage, but still can't access the data store. Any info is appreciated! 

Edit: @jkropko it's back up now

**Comment by njpolimeni** [comment]
I'm getting this same issue "Internal Server Error" now, as of March 19th 2024, when trying to request an API key.

**Comment by DustinHurtt** [comment]
Is there any other way to get an API key?  I'm currently being re-directed to a page that says Internal Server Error when I request a key at this site :  [https://www.propublica.org/datastore/api/propublica-congress-api](url)

**Comment by njpolimeni** [comment]
@dwillis Thanks for closing the issue; I see the website has now been updated to explain that the API is being discontinued. Do you know why this is?

**Comment by dwillis** [comment]
@Nicholas-Polimeni yes - ProPublica sent out the following email this week:


Dear users of the ProPublica Congress API,

As you have likely noticed, the API has been having intermittent issues this year. We apologize for the instability and any problems this may have caused you. While the API is back up for now, ultimately we have decided to discontinue supporting it, and will shut it down on July 5th. We hope these three months will be enough time to migrate your usage to other data sources, such as the Congress.gov API.

A little background may help: When it originated at The New York Times in 2009, the Congress API took information that was otherwise difficult to get in a machine-readable format, and centralized it. The API eventually moved here to ProPublica with its lead developer, Derek Willis, and later incorporated parts of the Sunlight Congress API. Derek left ProPublica in 2021 for the University of Maryland, and since then we have contracted with him to maintain both this API and the campaign finance API. 

We’re thankful for Derek’s help in keeping these services running, but it was never meant as a long-term solution. As the Congress.gov API provides much of the same data in a more reliable way, we believe it’s in the best interest of the API’s users to migrate to alternate sources.

We have already shut down new sign ups to the API, and we anticipate turning off the API itself on July 5 of this year. 

We will update you with more information in the coming weeks, including details on alternate sources of legislative data.

Thank you for your understanding, 
Ken Schwencke
Editor, News Applications
ProPublica
